### DISCUSSION TITLE
ws+tls 协议是不是快废了

### DISCUSSION THREAD

**Discussion by awsawsz** [XTLS/Xray-core#2394] [Q&A] [answered]
最近防火墙升级，好像进入泉州模式了，有没有大佬探究一下，表现如下：

- 新建域名直接reset，需要等待一段1分钟左右时间才可以访问。
- 自选IP行为被认为是不可取的，会被阻断。必须使用给定的IP
- 测速网站进行测速，页面被阻断后依然接收一段时间的数据。
- 日志频繁出现 443: wsarecv: An existing connection was forcibly closed by the remote host.

使用TLS WS VMESS/VLESS，其他未测试，
大佬分析一下什么原因，下面是不是进行全国推广？

**Comment by chika0801** [comment, marked as answer]
是的，一些收集见 https://github.com/chika0801/Xray-examples/blob/main/warning.md

**Reply by awsawsz** [reply]
QUIC可以通过CDN转发吗？

**Reply by chika0801** [reply]
不能 只有 gRPC ws  2个方式行

**Reply by RPRX** [reply]
~~为什么被 Unmark 了~~

**Reply by awsawsz** [reply]
我又加问了

**Comment by Jinnrry** [comment]
+1 我今年以来，无论是xray ws+nginx+tls  还是naiveproxy，都明显感觉到被阻断。同样的服务器，使用ws不通，但是换成xray+vision就能用
